readme.md

```diff
@@ -1,96 +1,64 @@
-# Bitbucket Server and Jenkins integration
+# Bitbucket Server integration
 [![Build Status](https://ci.jenkins.io/job/Plugins/job/atlassian-bitbucket/job/master/badge/icon)](https://ci.jenkins.io/job/Plugins/job/atlassian-bitbucket-integration/job/master/)
```

Comment:
```suggestion
[![Build Status](https://ci.jenkins.io/job/Plugins/job/atlassian-bitbucket-server-integration-plugin/job/master/badge/icon)](https://ci.jenkins.io/job/Plugins/job/atlassian-bitbucket-server-integration-plugin/job/master/)
```
The url was wrong, when the Readme was originally written we just put a placeholder in, we never updated it. The above should work :crossed_fingers: